how-to-release.md

```diff
@@ -84,65 +94,63 @@ for file in ./*.jar ; do shasum -a 1 $file > $file.sha1 ; done
 for file in ./*.jar ; do shasum -a 256 $file > $file.sha256 ; done
 ```
 
-*Note*: In the future, we might want to automate the generation of checksums via gradle.
+>**Note**: Generation of checksums via gradle is planned to be automated in the future releases.
 
-## Step 3: Upload the files to the pre-release draft
+### Step 3: Upload the files to the pre-release draft
 
 In the pre-release draft on GitHub, upload all the jar files (and its checksums) under `~/.m2/repository/io/pravega/pravega-connectors-flink_2.11/<RELEASE-VERSION>`. Follow the instructions on the page, it is straightforward.

```

Comment:
In the below statement:
Follow the instructions on the `page`, it is straightforward.???
which page??not clear

Reply:
> Follow the instructions that appears on the Github release page, it is straightforward.

Reply:
done